src/app/Scenes/SellWithArtsy/Components/SellWithArtsyRecentlySold.tsx

```diff
@@ -106,7 +107,7 @@ const RecentlySoldArtworksRail: React.FC<RecentlySoldArtworksRailProps> = ({
             onPress?.(item, index)
           }}
           // adding 20 to make sure the estimated section is not cut off when displayed in two lines
-          containerHeight={ARTWORK_RAIL_CARD_IMAGE_HEIGHT + PixelRatio.getFontScale() * 100 + 20}
+          containerHeight={ARTWORK_RAIL_CARD_IMAGE_HEIGHT + fontScale * 100 + fontScale * 20}
```

Comment:
Do you know where the 100 comes from?
```suggestion
          containerHeight={ARTWORK_RAIL_CARD_IMAGE_HEIGHT + fontScale * 120}
```

Reply:
100 is the height of the text component of 92px and 10px vertical margin

Actually, i need to change 100 to 102